GDJS/Runtime/runtimegame.ts

```diff
@@ -191,6 +191,7 @@ namespace gdjs {
     _sessionMetricsInitialized: boolean = false;
     _disableMetrics: boolean = false;
     _isPreview: boolean;
+    _displayMessageInPreview: boolean = false;
```

Comment:
This boolean is a bit weird, it acts as "show it once" but it's not super clear when reading its name. 
Do we really need it?